funtofem/interface/caps2fun/mesh_aim.py

```diff
@@ -0,0 +1,101 @@
+"""
+Written by Brian Burke and Sean Engelstad, Georgia Tech SMDO Lab, 2024.
+"""
+
+__all__ = ["MeshAim"]
+
+from .aflr_aim import Aflr3Aim, Aflr4Aim
+from .egads_aim import EgadsAim
+
+
+class MeshAim:
```

Comment:
Can you change this to `AeroMeshAim` as this class has volume and surface meshes? Or if not allow it to have no volume mesher.

Reply:
Or maybe it should be `AeroMeshAims` or something because it is both. Let's not overthink it though